pkg/cloudprovider/config.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2019 the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloudprovider
+
+import (
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+func ValidateConfigKeys(config map[string]string, validKeys ...string) error {
```

Comment:
This function is fairly simple, but maybe it makes sense to expose in the plugin `framework` package, so 3rd party plugins can also do validation. What do you think?

Reply:
Hmm. I think I'd rather have them vendor `pkg/cloudprovider` if they want to use it, since it's not related to the gRPC/go-plugin infrastructure for plugins.

Reply:
Fair enough.